<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Integrity checking</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <widget class="QListWidget" name="removed_items"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="added_items"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_added_items">
     <property name="text">
      <string>Added items</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_removed_items">
     <property name="text">
      <string>Removed items</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
